webroot/results/statistics.php

```diff
@@ -39,26 +39,28 @@ function showContent () {
     file_put_contents( 'generated/statistics.log', "$logMessage\n", FILE_APPEND );
     echo "<p>$logMessage</p>";
   }
-  
+
   #--- Show the cache
   echo file_get_contents( 'generated/statistics.cache' );
 }
 
 #----------------------------------------------------------------------
 function showResults () {
 #----------------------------------------------------------------------
-  
```

Comment:
I hear you. Anyway I really like that an editor can take care of this, so you don't leave those trailing white characters even without thinking about it. It's just an edge case that some old code has been touched by me =)